### DISCUSSION TITLE
GraphQL "Run GraphQL query"

### DISCUSSION THREAD

**Discussion by jaisonvm** [OrchardCMS/OrchardCore#13864] [Q&A]
I have enabled GraphQL feature and I can see it the admin panel but how can I enable “Run GraphQL query” feature
Please see the screenshot attached 
![GraphQL](https://github.com/OrchardCMS/OrchardCore/assets/136476039/13f2c0b2-7463-4b23-8531-1c57ec1827bf)

**Comment by lampersky** [comment]
this feature link is placed inside the configuration submenu:

![Screenshot_20230614_222458](https://github.com/OrchardCMS/OrchardCore/assets/6403130/bcd42664-497f-4e51-8cd1-bd7d536ff869)

**Reply by jaisonvm** [reply]
Thanks lampersky, but I am looking "Run GraphQL query" link under the Queries menu 
![2023-06-15_11h26_44](https://github.com/OrchardCMS/OrchardCore/assets/136476039/f31232cc-0322-4425-b5da-cf8faf8753a6)

**Reply by MarGraz** [reply]
@jaisonvm in the end did you found the `Run GraphQL query`? 

I'm also looking for the same feature, that is visible in this [video guide](https://www.youtube.com/watch?v=8SbW3TLNhF0), but in OC 1.8.3 seems that is not available. Maybe the video is too old.

Thank you

**Comment by gaojianzhuang** [comment]
I try to use the GraphQL query in the liquid template just like as below:
```
   {% graphql %}
   query {
     myQuery(fieldParam: "someValue") {
       resultField1
       resultField2
     }
   }
   {% endgraphql %}

   Result Field 1: {{ graphql_result.data.myQuery.resultField1 }}
   Result Field 2: {{ graphql_result.data.myQuery.resultField2 }}
```
I am also encountering the same problem. Anyone can help figure out the root cause? Thanks
